client/readme.md

```diff
@@ -11,15 +11,16 @@ The flow of data through the client is shown in this diagram:
    - the `message` service
    - the `chain` service.
 2. The engine reads channels and objectives from the `store`, and computes updates and side effects.
-3. The updates arre committed to the `store`.
-4. The side effects are sent on go channels to:
+3. The engine reads payments the `payment manager`, and computes updates and side effects.
```

Comment:
This sentence doesn't seem to scan right?

Reply:
I think I'm missing a `from`
```suggestion
3. The engine reads payments from the `payment manager`, and computes updates and side effects.
```

Reply:
Fixed in https://github.com/statechannels/go-nitro/pull/836/commits/1032e29cb979d9f897f934771fb7e72cfbd32bd0